Service Tasks/requesting-service-tasks-on-centurylink-cloud.md

```diff
@@ -1,31 +1,28 @@
 {{{
   "title": "Requesting Service Tasks on CenturyLink Cloud",
-  "date": "10-13-2014",
+  "date": "9-10-2015",
   "author": "Dana Bowlin",
   "attachments": [],
-  "contentIsHTML": true
+  "contentIsHTML": false
 }}}
 
-<h3>Description (goal/purpose)</h3>
-<p>Our team of network engineers has deep operational expertise to quickly and efficiently perform common platform activities. Just give us the details on your request – and within one business day, we'll reply with an estimate, along with a projected completion
-  date for the task. </p>
-<ul>
-  <li>Service task estimation and duties are performed during business hours, Monday-Friday, 9am-5pm Pacific Time.</li>
-  <li>The types of tasks we can perform on your behalf are listed on our website here:&nbsp;<a href="http://www.ctl.io/products/support/service-tasks">List of Service Tasks and Associated Costs</a>.</li>
-</ul>
-<h3><strong>Audience</strong></h3>
-<ul>
-  <li>All Customers</li>
-</ul>
-<h3>Prerequisites</h3>
-<ul>
-  <li>None</li>
-</ul>
-<h3>Detailed Steps to Request a Service Task</h3>
-<ol>
-  <li>Send email to <a href="mailto:ServiceTasks@tier3.com">ServiceTasks@tier3.com</a>. This will automatically create a ticket for the Service Tasks team to review. </li>
-  <li>You will get an acknowledgement that we have received your request. Requests are reviewed by our Service Tasks team daily.</li>
-  <li>From there, we work with you and your team to ensure the task is completed as requested. Updates on progress are provided via our ticketing system.</li>
-</ol>
+### Description (goal/purpose)
+Our team of network engineers has deep operational expertise to quickly and efficiently perform common platform activities. Just give us the details on your request – and within one business day, we'll reply with an estimate, along with a projected completion date for the task.
+* Service task estimation and duties are performed during business hours, Monday-Friday, 9am-5pm Pacific Time.
+* The types of tasks we can perform on your behalf are [listed on our website along with any associated fees.](//www.ctl.io/products/support/service-tasks)
+
+### Audience
+* All Customers
+
+### Prerequisites
+* None
+
+### Detailed Steps to Request a Service Task
+
+1. Send email to <ServiceTasks@ctl.io>. This will automatically create a ticket for the Service Tasks team to review.
```

Comment:
remove the brackets, or make it servicetasks@ctl.io an actual mailto link. that should resolve the error